pkg/gofr/datasource/file/ftp/fs.go

```diff
@@ -14,7 +14,11 @@ import (
 	file_interface "gofr.dev/pkg/gofr/datasource/file"
 )
 
-const appFtpStats = "app_ftp_stats"
+const (
+	appFtpStats   = "app_ftp_stats"
```

Comment:
```suggestion
	appFTPStats   = "app_ftp_stats"
```

It's an acronym, fixing an unexported constant won't cause problem